src/main/webapp/app/entities/select-num-of-results.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Dropdown, DropdownMenu, DropdownToggle, DropdownItem } from 'reactstrap';
+import { Translate } from 'react-jhipster';
+
+const SELECT_ARRAY = [10, 20, 50, 100];
+
+const SelectNumOfValues = props => {
+  const dropdownOpen = props.dropdownOpen;
```

Comment:
You can use destructuring

const { dropdownOpen, itemsPerPage, toggleSelect, selectFunc } = props;